### PR TITLE
Try to fix MacOS sound

### PR DESCRIPTION
**Pull request by StratusFearMe21**
This PR updates globkey and attempts to fix the terrible sound produced by MacOS when using PTT, see https://imgur.com/a/o9VdGPo

### CODE REVIEW

**Comment by StratusFearMe21** [comment]
I also noticed that the `shouldbreak` logic that was here before was removed by someone. I know I didn't in my previous PR, but this have something to do with this issue

**Comment by amitojsingh366** [comment]
> 
> 
> I also noticed that the `shouldbreak` logic that was here before was removed by someone. I know I didn't in my previous PR, but this have something to do with this issue

actually i was the one who removed the `shouldBreak` logic as i was the one who put it there but forgot to make it actually do the task of breaking the loop. instead i noticed `worker.terminate()` automatically breaks the loop for me so i didn't touch `shouldBreak` and forgot to remove it. basically `shouldBreak` was redundant.

File: baklava/src/utils/keybinds/worker.ts [hunk 1]
```diff
@@ -2,9 +2,13 @@ import { isMainThread, parentPort } from 'worker_threads';
 import globkey from 'globkey';
 
 if (!isMainThread) {
+    let prev_keys = ['', '']
```
Comment (amitojsingh366) [inline baklava/src/utils/keybinds/worker.ts:5]: wait this line wasn't here? how did typescript not throw a bazillion errors!?

**Review by amitojsingh366 — APPROVED** [review]

**Comment by StratusFearMe21** [comment]
okay

File: baklava/src/utils/keybinds/worker.ts (hunk 1 above)
Comment (StratusFearMe21) [reply baklava/src/utils/keybinds/worker.ts:5]: before, rust handled the prev_keys logic and all that stuff, so this update hands that over to TS.

Comment (amitojsingh366) [reply baklava/src/utils/keybinds/worker.ts:5]: ahh makes sense...

**Pull request merged**